apps/meteor/tests/unit/app/lib/server/apps/disableAppsWithAddonsCallback.spec.ts

```diff
@@ -107,13 +132,55 @@ describe('disableAppsWithAddonsCallback', () => {
 			getManager: () => mockManager,
 		} as unknown as AppServerOrchestrator;
 
-		const disableAppsWithAddonsCallback = makeDisableAppsWithAddonsCallback({ Apps: AppsMock });
+		await expect(
+			_disableAppsWithAddonsCallback(
+				{ Apps: AppsMock, sendMessagesToAdmins },
+				{ module: 'chat.rocket.test-addon', external: true, valid: false },
+			),
+		).to.not.eventually.be.rejected;
+
+		expect(AppsMock.installedApps).to.have.been.called();
+		expect(AppsMock.getManager()?.disable).to.have.been.called.once;
+		expect(AppsMock.getManager()?.disable).to.have.been.called.with('test-app-with-addon');
+	});
+
+	it('should not send messages to admins if no app was disabled', async () => {
```

Comment:
Does it make sense to have a test case for when messages are sent?

Reply:
Well, I guess it increases coverages :P

Reply:
Done